docs/source/en/glossary.mdx

```diff
@@ -73,14 +73,6 @@ by the tokenizer under the key "attention_mask":
 [[1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]]
 ```
 
-### autoencoding models 
-
-see [masked language modeling](#masked-language-modeling)
-
-### autoregressive models
-
-see [causal language modeling](#causal-language-modeling)
```

Comment:
I would keep those as these are sometimes used (a quick search in the repo shows quite a few instances).